### PR TITLE
Add storage deals section

### PR DESCRIPTION
**Pull request by mishmosh**
- Adds content to storage deals section
- Rename file & titles to store-making-storage-deals.md (including changes to config.js and home.vue)

@terichadbourne there is 1 todo to improve wording for a headline, would love any suggestions.

### CODE REVIEW

File: docs/how-to/store-making-storage-deals.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+In the discovery phase, clients identify the miners who might be able to store their data. They start by querying the chain for the list of registered miners, and sends each miner a request for its storage ask (short for *asking price*).
+
+The storage ask contains details such as price, duration, and available storage capacity. Clients use these details to choose their preferred miner.
+
+*Note: In the near future, 3rd party reputation systems will allow clients to check for geography, miner reputation, and other factors.*
+
+### Negotiating the storage deal
+
+Next, the client locks up sufficient funds in their wallet to pay for the deal. The client also calculates the unique piece commitment (CommP) for the data it intends to store, which is used for confirmation in later phases.
```
Comment (terichadbourne) [inline docs/how-to/store-making-storage-deals.md:20]: Would you like to say a little more here about what a CommP is?

File: docs/how-to/store-making-storage-deals.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Transferring data and beginning storage
+
+After the deal is published, the client begins transferring data to the miner. Upon receiving all of the data, the miner packs the data into a sector (TODO: define sector), seals it, and begins submitting proofs to the chain. Once the first proof makes it onto the chain, the client has certainty that the miner is correctly storing the data, and the storage deal is officially begun.
+
+### The rest of the deal’s lifetime (TODO: Find a better summary phrase)
+
+Throughout the deal, the miner submits ongoing proofs to the chain. Clients pay incrementally using the funds they previously locked up. If a proof is missing or delayed, the miner is penalized.
+
+Deals have a start and end time, specified by epoch (aka block height) in the client's original storage deal proposal. Once the blockchain reaches the ending block height, the storage deal is complete and miners have no more obligation. To continue storing their data, storage clients must make a new deal.
```
Comment (terichadbourne) [inline docs/how-to/store-making-storage-deals.md:34]:
```suggestion
Deals have a start and end time, specified by epoch (aka block height) in the client's original storage deal proposal. Once the blockchain reaches the ending block height, the storage deal is complete and miners have no further obligation. To continue storing their data, storage clients must make a new deal.
```

File: docs/how-to/store-making-storage-deals.md (hunk 1 above)
Comment (mishmosh) [reply docs/how-to/store-making-storage-deals.md:20]: Added a bit here.

**Comment by mishmosh** [comment]
* Applied all suggestions (thanks!)
* Fixed TODO
* Defined commP, as suggested

**Pull request merged**